src/components/Nav/Masthead/Masthead.tsx

```diff
@@ -12,7 +12,7 @@ class Masthead extends React.Component {
     return (
       <ToolbarNext>
         <ToolbarGroup>
-          <PfSpinner />
+          <PfSpinner isLoading={false} />
```

Comment:
Why are you passing isLoading when this value is from redux ? 

```ts
const mapStateToProps = (state: KialiAppState) => ({
  isLoading: state.globalState.loadingCounter > 0
});
```

Reply:
Good catch, there's something funky going on with properties coming from redux (they can not be marked as required because they might not be there).